demos/index.html

```diff
@@ -518,101 +518,52 @@ <h3>
         Phase 2: Accessible Actions / Input Events
       </h3>
       <p>
-        Listen to input events directly from assistive technology.
+        To preserve the privacy of assistive technology users, 
+        events from assistive technology will typically cause a synthesised DOM event to be triggered:
       </p>
       <pre>
-    slider.addEventListener("accessibleincrement", function(event) {
-      console.log("Got Increment from assistive technology");
+    slider.addEventListener("keydown", function(event) {
+      // Keydown event may be generated via keyboard or via assistive technology gesture
+      console.log("Got keydown event");
+      switch (event.code) {
+        case "ArrowUp":
+          customSlider.value += 1;
+          return;
+        case "ArrowDown":
+          customSlider.value -= 1;
+          return;
+      }
     });
-  </pre>
-      <p>
-        Possible events include:
-      </p>
-      <pre>
-    "accessiblecontextmenu"
-    "accessibledecrement"
-    "accessibledismiss"
-    "accessiblefocus"
-    "accessibleincrement"
-    "accessiblescrollintoview"
-    "accessibleselect"
-    "accessiblesetvalue"
-    "accessibleclick"
-  </pre>
     </article>
 
     <article>
       <h3>
-        Phase 2 Use cases include...
-      </h3>
-      <ul>
-        <li>Focus events in composite controls, e.g. when the AT wants to focus an option or cell, but the web app wants to focus
-          the listbox or grid and use aria-activedescendant.
-          <li>Events that don't have any equivalent on the web now, like
-            <code>"increment"</code>,
-            <code>"decrement"</code>, or
-            <code>"dismiss"</code>.
-            <li>Capturing events on virtual nodes intended entirely for AT (more in Phase 3).
-      </ul>
-    </article>
-
-    <article>
-      <h3>
-        Phase 2 and privacy
+        Phase 2: Accessible Actions / Input Events
       </h3>
       <p>
-        If an AOM event listener is ever called, the web page *definitively knows* that the user is running AT.
-      </p>
-      <p>
-        Some users may not be comfortable revealing this, due to concerns of discrimination, or of being offered an undesired alternate
-        interface.
+        New input event types capturing common patterns will also be added, 
+        and where supported will be triggered by user interactions which trigger 
```

Comment:
nit: this sentence is hard to follow, can it be reworded at all?

Reply:
Thanks for catching :) Reworded, and I added another paragraph for an example.